omero/developers/JavaGateway.txt

```diff
@@ -0,0 +1,20 @@
+Java Gateway
+------------
+
+The Java :javadoc:`Gateway <omero/gateway/Gateway.html>` is a
+wrapper around the :zerocdoc:`Ice Java language mapping
+<display/Ice/Hello+World+Application>` and the :doc:`Modules/Api` which makes
+it easier to interact with an OMERO server in Java.
+
+The :javadoc:`Gateway <omero/gateway/Gateway.html>` is the central object
+for maintaining the connection to the server. 
+See :ref:`gatewayconnect`
```

Comment:
No full stop at the end of this sentence and actually I'm not sure this needs to be a separate sentence, maybe just "...to the server, see 'Connect to OMERO' for further information.